db_client/alembic/versions/0013_families_and_collections.py

```diff
@@ -234,6 +234,7 @@ def upgrade():
             ["event_type_name"],
             ["family_event_type.name"],
             name=op.f("fk_family_event__event_type_name__family_event_type"),
+            ondelete="CASCADE",
```

Comment:
Shouldn't this be a new migration to alter the table before we add the delete migration as opposed to editing an old migration? 
E.g. 
- 0038 update table to ondelete cascade 
- 0039 delete table

Reply:
I think this was actually leftover from a merge and its snuck in - great spot. Lemme remove